scripts/deploy.ps1

```diff
@@ -12,6 +12,14 @@ Compress-Archive -Path src\backend\index.js -DestinationPath $BuildDirectoryName
 Write-Host "Uploading notification function zip to S3 bucket"
 aws s3 cp $BuildDirectoryName\function.zip s3://filcel-deployments/function.zip
 
-Write-Host "Creating stack using CloudFormation"
+aws cloudformation describe-stacks --stack-name filcel-stack-dev
+Write-Host $LASTEXITCODE
 
-aws cloudformation create-stack --stack-name filcel-stack-dev --template-body file://src/backend/template.yml --parameters ParameterKey=Environment,ParameterValue=dev --capabilities CAPABILITY_NAMED_IAM
\ No newline at end of file
+if ($LASTEXITCODE -eq 0)
+{
```

Comment:
Stack name is present in the script in multiple places. It is prone to typos. Consider using a PS variable for the stack name.